App.tsx

```diff
@@ -1,20 +1,114 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { StatusBar } from "expo-status-bar";
+import { useState } from "react";
+import {
+	StyleSheet,
+	View,
+	FlatList,
+	Image,
+	TouchableOpacity,
+	TextInput,
+} from "react-native";
+import { ImageData } from "./ImageData";
+
+const imageData: ImageData[] = [];
+for (let i = 1; i < 70; i++) {
+	imageData.push({ id: i, url: `https://picsum.photos/id/${i}/200` });
+}
 
 export default function App() {
-  return (
-    <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
-  );
+	const [displayLgImg, setDisplayLgImg] = useState(false);
+	const [featuredImgUrl, setFeaturedImgUrl] = useState("");
+	const [searchQuery, setSearchQuery] = useState("");
+	const [activeImages, setActiveImages] = useState(imageData);
+
+	const search = (query: string) => {
+		const filteredContacts = imageData.filter((image) =>
+			image.id.toString().toLowerCase().includes(query.toLowerCase())
+		);
+		setActiveImages(filteredContacts);
+		setSearchQuery(query);
+	};
+
+	const showLgImg = (source: string) => {
```

Comment:
no need to abbreviate. I'd also try to keep naming consistent when referencing the large/featured image.